javacutil/src/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -247,6 +247,18 @@ public static VariableElement findFieldInType(TypeElement type, String name) {
         return null;
     }
 
+    /**
+     * Returns the elements of the fields whose simple names are {@code names} and are declared in
```

Comment:
Do you mean should this method only return non-private fields?  Or are you asking if the method as implemented only returns non-private fields?